lib/bundle/moduleWrapper.js

```diff
@@ -59,7 +60,10 @@ const isAnonymousAmd = (moduleContents) =>
  * @param {string} moduleContents Contents of the module to wrap.
  */
 const wrapAnonymousAmd = (moduleName, moduleContents) =>
-    moduleContents.replace(/define\s*\(/m, `define('${moduleName}', `);
+    moduleContents.replace(
+        /(^|\s+|;)define\s*\(/m,
+        `\n;define('${moduleName}', `
```

Comment:
This line is causing a problem when adding the `;` when we have a webpack module loaded like so:

```
(function webpackUniversalModuleDefinition(root, factory) {
    if(typeof exports === 'object' && typeof module === 'object')
        module.exports = factory(require("jquery"));
    else if(typeof define === 'function' && define.amd)
        define(["jquery"], factory);
    else if(typeof exports === 'object')
        exports["module.name"] = factory(require("jquery"));
    else
        ...
```

We end up with this occurring:

```diff
    (function webpackUniversalModuleDefinition(root, factory) {
    if(typeof exports === 'object' && typeof module === 'object')
        module.exports = factory(require("jquery"));
    else if(typeof define === 'function' && define.amd)
-        define('module/path/here', ["jquery"], factory);
+;define('module/path/here, ["jquery"], factory);
    else if(typeof exports === 'object')
        exports["module.name"] = factory(require("jquery"));
    else
        ...
```

Which in turn produces the below error in the bundled JS due to it being invalid JS:

```
Uncaught SyntaxError: Unexpected token 'else' (at bundle-common.js:42559:2)
```

Reply:
Thanks for the feedback. I didn't consider the case where you're missing braces. I guess it's been a while since I worked with Python as my day job. I'll update the pull request to account for this case.

Reply:
@adamj88 Please can you test again?